packages/admin-ui/ui/src/utils/currencies.ts

```diff
@@ -652,6 +652,15 @@ export const currencies: CurrenciesType = {
     code: "MMK",
     name_plural: "Myanma kyats",
   },
+  MNT: {
+    symbol: "MNT",
+    name: "Mongolian Tugrik",
+    symbol_native: "₮",
+    decimal_digits: 0,
```

Comment:
> question: [According to other platforms](https://docs.adyen.com/development-resources/currency-codes/), this currency should have two decimal digits. What made you specify 0?

Sorry, you're right. I should have looked it up when I made the commit. I just instinctively thought it was 0 because in Mongolia, we never really use decimal digits except for banks.